other/material/import_material_from_jarvis_db_entry.ipynb

```diff
@@ -1,137 +1,749 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    def atoms_to_job(name='interface-670-664', atoms=[],workflow=[]):
</pre><p>What would be a good signature for this function eventually?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>